PowerShell/ScubaGear/Modules/Providers/ExportEXOProvider.psm1

```diff
@@ -167,10 +168,12 @@ function Invoke-RobustDnsTxt {
     [CmdletBinding()]
     param (
         [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
         [string]
         $Qname,
 
         [Parameter(Mandatory=$false)]
+        [ValidateNotNullOrEmpty()]
         [string]
```

Comment:
MaxValue is most likely to large for a reasonable re-try value.  But if value is negative it will never try.  
```suggestion
        [ValidateRange(1, [int]::MaxValue)]
        [int]
```